services/outputhost/ackmanager.go

```diff
@@ -297,9 +266,9 @@ func (ackMgr *ackManager) updateAckLevel() {
 			ExtentUUID:         common.StringPtr(ackMgr.extUUID),
 			ConnectedStoreUUID: common.StringPtr(*ackMgr.connectedStoreUUID),
 			AckLevelAddress:    common.Int64Ptr(int64(ackMgr.ackLevelAddr)),
-			AckLevelSeqNo:      common.Int64Ptr(int64(ackMgr.levelOffset + ackMgr.ackLevel)), // levelOffset adjusts the level according to what was returned from getAddressFromTimestamp
+			AckLevelSeqNo:      common.Int64Ptr(int64(ackMgr.addrs[ackMgr.ackLevel].seq)),
```

Comment:
if you do the above assignments, then this will be just 

AckLevelSeqNo: common.Int64Ptr(ackMgr.ackLevelStoreSeq)